test/unit/webdriver/webdriver_test.py

```diff
@@ -82,3 +82,17 @@ def test_create_session_forceMjsonwp(self):
 
         assert driver.session_id == 'session-id'
         assert driver.w3c is False
+
+    @httpretty.activate
+    def test_a_number_of_commands_w3c_android(self):
+        """Track how many commands defined for Android W3C"""
+        driver = android_w3c_driver()
+        # pylint: disable=W0212
+        assert len(driver.command_executor._commands.keys()) == 174
```

Comment:
I would say it is an extra effort to always update this number when a new command is added/removed

Reply:
Definitely, I thought so. Will remove it